templates/master.yaml.tmpl

```diff
@@ -805,7 +805,7 @@ systemd:
           -e VAULT_ADDR\
           -e CLUSTER_NAME\
           {{.DockerRegistry}}/giantswarm/vault:1.6.1\
-          login -method=aws role=$${CLUSTER_NAME}-master -format=json | jq -r .auth.client_token);\
+          login -method=aws role=$${CLUSTER_NAME}-master region={{ .AWSRegion }} -format=json | jq -r .auth.client_token);\
```

Comment:
Looks ok but how was it working so far? Also may be a step backwards when it comes to multi-region support if we are planning to ever do that.

Reply:
It is a new service. We didn't have it before. Also multi-region support has nothing to do with this service.

Reply:
Do you know what is this region used for? I quickly checked https://www.vaultproject.io/docs/auth/aws but it didn't make much sense to me. I thought IAM is global service.

Reply:
China uses a different STS endpoint, therefore, requires setting a corresponding region explicitly

Reply:
And there is example on the page you've referenced

```
The region used defaults to us-east-1, but you can specify a custom region like so:

$ vault login -method=aws region=us-west-2 role=dev-role-iam

When using a custom region, be sure the designated region corresponds to that of the STS endpoint you're using.
```